static/css/modules/_ubuntu_patterns_footer.scss

```diff
@@ -0,0 +1,217 @@
+      a:visited {
+        color: $color-mid-dark;
+        font-weight: normal;
+        padding-left: 10px;
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          color: $color-dark;
+          padding-left: 0;
+        }
+      }
+                  
+      a::after {
+        content: ""; /* hide the default > */
+      }
+      
+      /// open and close the primary/secondary navs
+      + .second-level-nav {
+        display: none;
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          display: block;
+        }
+      }
+      
+      &.active {
+        background-image: url("#{$asset-server-url}43e2b367-arrow_up_9fa097.png");
+  
+        @media only screen and (min-width: $navigation-threshold) {
+          background: none;
+        }
+        
+        + .second-level-nav {
+          display: block;
+        }
+      }
+    }
+    
+    &__link {
+      &:hover {
+        color: $color-dark;
+      }
+    }
+    
+    .top-link {
+      @include footer-title;
+      background-image: url("#{$asset-server-url}e4a5a8c0-picto-upload-warmgrey.svg");
+      background-position: 10px 5px;
+      background-repeat: no-repeat;
+      background-size: 14px 14px;
+      border-bottom: 1px solid $color-dark;
+      display: block;
+      font-size: 1em;
+      left: auto;
+      padding: 5px 0 10px 30px;
+      position: relative;
+      
+      @media only screen and (min-width: $navigation-threshold) {
+        display: none;
+      }
+    }
+  }
+  
+  // second level nav - HTML for this can't be changed as it's used in the main nav also
+  .second-level-nav {
+    margin: 0;
+    
+    li {
+      margin-bottom: 0;
+      
+      a {
+        border-top: 1px solid $color-mid-light;
+        color: $color-dark;
+        display: block;
+        font-size: .8125em;
+        padding: 10px;
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          border: 0;
+          font-size: 1em;
+          padding: 0 0 5px 0;
+        }
+        
+        @media only screen and (min-width: $breakpoint-large) {
+          font-size: .8125em;
+        }
+      }
+    }
+  }
+  
+  .p-footer--secondary {
+    padding: 20px 10px 0;
+    
+    @media only screen and (min-width: $navigation-threshold) {
+      padding: 0; 
+    }
+    
+    a:hover {
+      color: currentColor;
+    }
+    
+    &__content {
+      font-size: 1em;
+    }
+    
+    .button--primary {
+      color: $color-dark;
+      background-color: $white;
+      border: 1px solid $color-mid-light;
+      padding: 6px 18px;
+  
+      &:hover {
+        background-color: #f7f7f7;
+      }
+    } 
+    
+    .p-inline-list--small {
+      .p-inline-list__item {
+        display: block;
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          display: inline-block;  
+        }
+      }
+    }
+  }
+
+  // social icons in footer
+  .p-inline-list-icons {
```

Comment:
Could you use inline-list? https://docs.vanillaframework.io/en/patterns/lists#inline-list